todo_app/trello_board_actions.py

```diff
@@ -0,0 +1,102 @@
+import os
+import requests
+from todo_app.item import Item
+
+class Trello_Board_Actions:
+    " Trello util to get lists on a board, get cards and their statuses, update/delete a card"    
+
+    def __init__(self):
+        self.auth = {'key':os.getenv("SECRET_KEY"), 'token':os.getenv("TOKEN"), 'cards':"open"}
+        self.url= "https://api.trello.com/1"
+        self.headers = {
+            'type': "type",
+            'content-type': "application/json"
+                }
+
+    def getBoardLists(self):
```

Comment:
Naming convention is snake_case for variable and function names; PascalCase for class names